Functional.py

```diff
@@ -0,0 +1,67 @@
+def sequential_map(*args):
+    """
+    Returns list of values with sequential application of provided functions
+    @ param values - list of needed values
+    @ param funcs - tupple of applied functions
+    @ return - processed list
+    """
+    *funcs, values = args
+    combo_func = func_chain(*funcs)
+    values = [_ for _ in map(combo_func, values)]
```

Comment:
`_` обычно используют в `for`, если это значение нам не важно. Здесь лучше вставить какое-нибудь абстрактное `i`
```python
values = [i for i in map(combo_func, values)]
```
а ещё лучше вот так
```suggestion
    values = list(map(combo_func, values)
```